llvm/test/MC/Disassembler/X86/apx/imul.txt

```diff
@@ -21,6 +21,11 @@
 # INTEL: {evex}	imul	r9, r9, 123
 0x62,0x54,0xfc,0x08,0x6b,0xc9,0x7b
 
+# iOSize=64 when EVEX.W = 1 and EVEX.pp = 01
```

Comment:
Add a separate test for this and check more instructions like llvm/test/MC/Disassembler/X86/apx/reverse-encoding.txt

Reply:
[1e5d7d7](https://github.com/llvm/llvm-project/pull/103816/commits/1e5d7d7646246a394e27f642fa7df01b0c6f254b)